src/ApprenticeAan/SFA.DAS.ApprenticeAan.Api/Controllers/ProfilesController.cs

```diff
@@ -0,0 +1,28 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.ApprenticeAan.Application.Profiles.Queries.GetProfilesByUserType;
+
+namespace SFA.DAS.ApprenticeAan.Api.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class ProfilesController : Controller
+    {
+        private readonly IMediator _mediator;
+
+        public ProfilesController(IMediator mediator) => _mediator = mediator;
+
+        /// <summary>
+        /// Get list of profiles by user type
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet]
+        [Route("{userType}")]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(GetProfilesByUserTypeQueryResult), StatusCodes.Status200OK)]
```

Comment:
you can add
        [ProducesResponseType(StatusCodes.Status500InternalServerError)]

Reply:
This is not required.